tools/generate_games_list.js

```diff
@@ -21,12 +21,17 @@ sortedGamesIds.forEach(key => {
 })
 
 let generated = ''
-generated += '| GameDig Type ID | Name | See Also\n'
-generated += '|---|---|---\n'
+generated += '| GameDig Type ID | Old ID | Name | See Also\n'
```

Comment:
Hmmm, i think we shouldn't add the Old ID as a column in the `GAMES_LIST` file.
These ids will eventually be removed in a future version and I think by keeping them would incentivese users to keep them or for new users to use the olds ids as they would appear to be more pleasing (related to the aliasing idea you gave). Thoughts?

Having a markdown file dedicated to migrating from v4 to v5 would be ideal and these `old vs new` ids would be perfect there.

Reply:
Oh yeah, we've discussed this. Let me create this document as well.

If the idea is to eventually remove them, then yeah, let's not promote them. But what's the downside of having them as ids that used to work?

Reply:
Yes, that's not a downside, i just meant to do not include them in this file.